app/presenters/vendor_api/base.rb

```diff
@@ -53,7 +53,14 @@ def version_available_in_environment?(version)
     end
 
     def cache_key(model, api_version, suffixes = {})
-      CacheKey.generate("#{api_version}_#{model.cache_key_with_version}#{suffixes.hash}")
+      suffixes_name = suffixes.values.compact_blank.join('-')
```

Comment:
This looks good to me, I'm wondering if we need to put a comment or something saying this will not reflect a change to the response in production as we don't set a new key when we deploy anymore.

Reply:
We could increment a counter in the class that we updated, and update it when we add a new property to the schema. But the cache expires in 1 day, adding a new property is rarely going to need to be available within 24hrs of the update

```ruby
class ApplicationPresenter
  def cache_version
    3
  end
end
```